src/main/scala/org/constellation/Statistics.scala

```diff
@@ -0,0 +1,76 @@
+package org.constellation
+
+import cats.effect.{Blocker, ExitCode, IO, IOApp, Resource}
+import com.amazonaws.auth.DefaultAWSCredentialsProviderChain
+import com.amazonaws.services.s3.AmazonS3ClientBuilder
+import com.amazonaws.util.IOUtils
+import fs2.Stream
+import fs2.io.{stdin, stdinUtf8}
+import org.constellation.schema.snapshot.SnapshotInfo
+import org.constellation.serializer.KryoSerializer
+
+import scala.concurrent.ExecutionContext
+import scala.io.StdIn.readLong
+
+object Statistics extends IOApp {
+
+  val s3BucketName = "constellationlabs-block-explorer-mainnet"
+  val s3Region = "us-west-1"
+  val height = "1114830"
+  val hash = "6160db2d7582e0a6fd20e0a1593cbb28ffea4b8af00ecf3b1788b7f35cd5c665"
+
+  override def run(args: List[String]): IO[ExitCode] =
+    main.compile.drain.as(ExitCode.Success)
+
+  val s3Client = Stream.eval {
+    IO {
+      AmazonS3ClientBuilder
+        .standard()
+        .withRegion(s3Region)
+        .withCredentials(new DefaultAWSCredentialsProviderChain())
+        .build()
+    }
+  }
```

Comment:
I would use bracket there:

```scala
val s3Client = Stream.bracket(
  IO(
    AmazonS3ClientBuilder
      .standard()
      .withRegion(s3Region)
      .withCredentials(new DefaultAWSCredentialsProviderChain())
      .build()
  )
)(c => IO(c.shutdown()))
```

Lack of shutdown won't have any consequences there but it is always good to shutdown client.

Reply:
It shouldn't be a `val` then but created and passed to the service